src/main/java/ch/wisv/events/core/model/event/Event.java

```diff
@@ -227,4 +227,13 @@ public int getReserved() {
     private double calcProgress(double reserved) {
         return Math.min(Math.round(((reserved / (double) this.target) * 100.d) * 100.d) / 100.d, 100.d);
     }
+
+    /**
+     * Get all products of this event
+     *
+     * @return List<Product>
+     */
+    public List<Product> getProducts(){
```

Comment:
This class has a `@Data` annotation, so boilerplate getters like this are already generated

Reply:
Removed